src/main/java/net/imagej/ops/filter/bilateral/DefaultBilateral.java

```diff
@@ -48,12 +48,12 @@
 import org.scijava.plugin.Plugin;
 
 /**
- * Performs a bilateral filter on an image. The parameter m_SigmaS refers to the
+ * Performs a bilateral filter on an image. The sigmaS parameter refers to the
  * spatial smoothing parameter; the greater the sigma, the smoother the image.
- * The parameter m_SigmaR refers to the range smoothing parameter. The greater
- * the sigma, the greater the effect of intensity differences. The parameter
- * m_radius refers to the square that is considered when doing the filter on
- * each individual picture.
+ * The sigmaR parameter refers to the range smoothing parameter. The greater the
```

Comment:
Instead of documenting parameters in the main section like this, you can put javadoc on the actual parameters below. This keeps the descriptions associated directly with the params, such that if a param is changed or removed later, the documentation will be less likely to be overlooked.

Reply:
Should be fixed!